datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -262,13 +262,38 @@ func (f *ChainDataFetcher) stopRangeFetching() error {
 	return nil
 }
 
+func (f *ChainDataFetcher) retryMakeChainEvent(blockNumber uint64) (blockchain.ChainEvent, error) {
+	var (
+		retryMax      = 100
+		retryInterval = 1 * time.Second
+	)
+	ev, err := f.makeChainEvent(blockNumber)
+	for retryCount := 1; err != nil; retryCount++ {
+		select {
+		case <-f.stopCh:
+			return ev, err
+		default:
+			logger.Warn("retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", retryCount, "retryMax", retryMax, "err", err)
+			if retryCount >= retryMax {
+				logger.Error("the max retry exceeded")
```

Comment:
* Shouldn't this case be paused?
* When this happens, it would be hard to check the log file. How about making this log every 100 retries, for example?

Reply:
- As I mentioned in the description, I wanted to retry the `makeChainEvent` infinitely. So, the code is not returned, but it leaves error log after the threshold.
- Do you think that this is too verbose? I think it is fine since the interval is 1 second. How about increasing log level instead of the `Warn` after `retryMax` instead?

Reply:
I don't think the changing the logging level doesn't affect much.
I think it is verbose to print every second. It would be the only log we could find when this happens. Could you please change it to log every 10 seconds at least.

Reply:
@winnie-byun I just updated the log frequencies. It leaves warning logs for the number of max retry. After that it leaves an error log every 100 retries.